lib/plug_rails_cookie_session_store.ex

```diff
@@ -134,17 +142,18 @@ defmodule PlugRailsCookieSessionStore do
   defp generate_key(secret, nil, _), do: secret
   defp generate_key(secret, key, key_opts), do: KeyGenerator.generate(secret, key, key_opts)
 
-  defp validate_secret_key_base(nil), do:
-    raise(ArgumentError, "cookie store expects conn.secret_key_base to be set")
-  defp validate_secret_key_base(secret_key_base) when byte_size(secret_key_base) < 64, do:
-    raise(ArgumentError, "cookie store expects conn.secret_key_base to be at least 64 bytes")
-  defp validate_secret_key_base(secret_key_base), do:
-    secret_key_base
+  defp validate_secret_key_base(nil),
+    do: raise(ArgumentError, "cookie store expects conn.secret_key_base to be set")
+
+  defp validate_secret_key_base(secret_key_base) when byte_size(secret_key_base) < 32,
+    do: raise(ArgumentError, "cookie store expects conn.secret_key_base to be at least 32 bytes")
```

Comment:
Since Rails 5.2 `secret_key_base` is generated automatically in TEST and DEV. This is simply derived as a MD5 hash of the application's name and its length is 32 bytes.